patches/bank3e.py

```diff
@@ -79,3 +80,5 @@ def addBank3E(rom):
 
     # Put 20 rupees in all owls by default.
     rom.patch(0x3E, 0x3B16, "00" * 0x316, "1C" * 0x316)
+
+    rom.patch(0x3E, 0x2F00, "00" * 0x10, binascii.hexlify(seed))
```

Comment:
The seed doesn't have to be 16 bytes if manually entered, so "00" * len(seed) would be a bit more future proof. Even if reading back is slightly wrong then.

Reply:
Oh, good call.  Hard coding that length is a bad idea even without the manual seed entry.  Done.